mlflow/pipelines/classification/v1/pipeline.py

```diff
@@ -0,0 +1,188 @@
+"""
+.. _mlflow-classification-pipeline:
+
+The MLflow Classification Pipeline is an MLflow Pipeline for developing classification models.
```

Comment:
Do our docs mention that only binary classification is supported initially? 

Also, what's the behavior if a user passes in a dataset with > 2 classes? Do we throw or build a one-vs-rest classifier? We should probably make sure that we do one of these.

Reply:
Good question. We would built a one-vs-rest classifier. What do you think desired behavior should be here?

Reply:
One-vs-rest sounds great! Can we make sure to document this behavior?